packages/components/pages/f-checkout/src/components/Checkout.vue

```diff
@@ -798,6 +817,28 @@ export default {
             this.handleEventLogging('CheckoutValidationError', validationState, { ...expandedData, validationState });
         },
 
+        resetLoadingState () {
+            this.isLoading = false;
+            this.shouldShowSpinner = false;
+        },
+
+        startSpinnerCountdown () {
+            setTimeout(() => {
+                if (this.isLoading) {
+                    this.shouldShowSpinner = true;
+                }
+            }, this.spinnerTimeout);
+        },
+
+        /**
+         * Sets the submitting state of the Checkout form. When true a spinner is displayed on the submit button
+         * This is done to allow us to test the setting of this value and ensure it is called with the correct value in the correct order.
+         * @param  {boolean} isFormSubmitting  - whether the form should be in a submitting state or not.
+         */
+        setSubmittingState (isFormSubmitting) {
+            this.isFormSubmitting = isFormSubmitting;
+        },
+
```

Comment:
This function doesn't match up with #1404 and doesn't seem to be referenced anywhere else in the PR - I'm assuming this was left in from another changeset?

Reply:
Ah yes, this should be in a different component now. Thanks! I'll remove it

Reply:
Removed Thanks!